modin/experimental/core/execution/native/implementations/hdk_on_native/expr.py

```diff
@@ -1107,6 +1108,9 @@ def _fold_invert(self) -> Union["OpExpr", LiteralExpr]:
                 return OpExpr("IS NOT NULL", op.operands, op._dtype)
             if op.op == "IS NOT NULL":
                 return OpExpr("IS NULL", op.operands, op._dtype)
+            if op.op == "CASE":
```

Comment:
Again, not related to arrow exporting changes?

Reply:
Yes, it was a test failure fix. I don't remember which one.